resources/examples/spidacalc/demos/framing-demo/calcdemo.js

```diff
@@ -1,164 +0,0 @@
-
-window.onload = function() {
```

Comment:
@spidaMikeFord Do we need an example for adding input assemblies through the API?

Reply:
We do... I have some I used for testing that I will attach.